luma/lcd/device.py

```diff
@@ -757,3 +760,234 @@ def contrast(self, value):
         """
         assert(0 <= value <= 255)
         self.command(0x81, value >> 2)
+
+
+class hd44780(parallel_device, character):
+    """
+    Driver for a HD44780 style LCD display.  This class provides a ``text``
+    property which can be used to set and get a text value, which will be
+    rendered to the display's screen using the display's built-in font.
+
+    :param serial_interface: The serial interface (usually a
+        :py:class:`luma.core.interface.serial.parallel` instance) to delegate
+        sending data and commands through.
+    :param width: The number of characters that can be displayed on a line.
+    :type width: int
+    :param height: The number of lines the display supports.
+    :type height: int
+    :param undefined: character to use if a requested character is not in the
+        font tables
+    :type undefined: str
+    :param selected_font: the font table appropriate for the model of display you
+        are using.  The hd44780 normally comes in a version with font A00 (ENGLISH_JAPANESE) or A02 (ENGLISH_EUROPEAN).  You can provide either
+        the name ('A00' or 'A02') or the number (0 for 'A00', 1 for 'A02') for
+        the font your display contains.
+    :type selected_font: int or str
+    :param exec_time: Time in seconds to wait for a command to complete.
+        Default is 50 μs (1e-6 * 50) which typically is long enough for commands
+        to finish.  If your display is not working correctly, you may want to
+        try increasing the exec_time delay.
+    :type exec_time: float
+    :param framebuffer: Framebuffering strategy, currently values of
+        ``diff_to_previous`` or ``full_frame`` are only supported.
+    :type framebuffer: str
+
+    To place text on the display, simply assign the text to the 'text'
```

Comment:
instead of 'text' use double backticks for ```text```

Reply:
Changed as suggested